liquibase-core/src/main/java/liquibase/snapshot/DatabaseSnapshot.java

```diff
@@ -447,9 +447,9 @@ private Object replaceObject(Object fieldValue) throws DatabaseException, Invali
             List<Object> newValues = new ArrayList<>();
             while (fieldValueIterator.hasNext()) {
                 Object obj = fieldValueIterator.next();
-                if ((fieldValue instanceof DatabaseObject) && !snapshotControl.shouldInclude(((DatabaseObject)
-                        fieldValue).getClass())) {
-                    return fieldValue;
+                if ((fieldValue instanceof DatabaseObject)) {
```

Comment:
We have changed the comparison here and not returning anything here now. Is that intended?